tinsel/src/commonMain/kotlin/com/mattprecious/tinsel/Tinsel.kt

```diff
@@ -1,9 +1,9 @@
 package com.mattprecious.tinsel
 
 object Tinsel {
-  fun render(nodes: List<Node>): String {
-    if (nodes.isEmpty()) return ""
-    return buildString { appendNodes(nodes) }
+  fun render(tree: Tree): String {
```

Comment:
Probably should become an extension next

Reply:
Ah, yep.